### PR TITLE
Create wrapper for json module

### PR DESCRIPTION
**Pull request by davydovanton**
/cc @hanami/contributors, @joneslee85 and @jodosha

### CODE REVIEW

File: lib/hanami/logger.rb [hunk 1]
```diff
@@ -156,7 +157,7 @@ class JSONFormatter < Formatter
       # @since x.x.x
       # @api private
       def _format(hash)
-        JSON.generate(hash)
+        Hanami::Utils::Json.dump(hash)
```
Comment (joneslee85) [inline lib/hanami/logger.rb]: should upcase the Json keyword?

**Comment by joneslee85** [comment]
LGTM

File: lib/hanami/logger.rb (hunk 1 above)
Comment (davydovanton) [inline lib/hanami/logger.rb]: I think no. I want to use hanami json wrapper here, that's why I use downcased `json` keyword

**Comment by jodosha** [comment]
@davydovanton Thank you. I just merged this!

**Comment by davydovanton** [comment]
Thanks for merge! :tada:

**Pull request merged**